doc/classes/RDFramebufferPass.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="RDFramebufferPass" inherits="RefCounted" version="4.0">
+	<brief_description>
+		Framebuffer pass attachment description.
+	</brief_description>
+	<description>
+		This class contains the list of attachment descriptions for a framebuffer pass. Each points with an index to a previously supplied list of texture attachments.
+	Multipass framebuffers can optimize some configurations in mobile, on desktop they provide little to no advantage.
```

Comment:
I would mention TBDR somewhere here as the reason for optimization, while it's mostly used in mobile it's a better starting point for anyone trying to understand why this is useful. Soemthing like `Multipass framebuffers can optimize some configurations in Tile Based Deferred Rendering GPUs (moslty mobile, on desktop they provide little to no advantage).`